R/pkg/R/mllib.R

```diff
@@ -695,17 +695,15 @@ setMethod("predict", signature(object = "KMeansModel"),
 #' @note spark.mlp since 2.1.0
 setMethod("spark.mlp", signature(data = "SparkDataFrame"),
           function(data, layers, blockSize = 128, solver = "l-bfgs", maxIter = 100,
-                   tol = 1E-6, stepSize = 0.03, seed = 0x7FFFFFFF) {
+                   tol = 1E-6, stepSize = 0.03, seed = "") {
```

Comment:
@felixcheung 
Oh...here is the problem, if the parameter is integer passed in, some integer exceed MAX_32bit_INT may turn into numeric, for example 1000000000000000L became 1e+15.
so how to resolve this problem properly ?